config/nginx.conf.erb

```diff
@@ -26,6 +26,11 @@ http {
 	#Must read the body in 5 seconds.
 	client_body_timeout 5;
 
+	include config/00_real_ip_cloudflare_00.conf;
+	# Heroku Router
+	set_real_ip_from 10.0.0.0/8;
```

Comment:
🤔

Reply:
It's the Heroku Router's ips that we receive in REMOTE_ADDR. We need to allow them otherwise the real address will be always the heroku's one.